CHANGELOG.md

```diff
@@ -11,8 +11,11 @@
 * Add option to add import statements at the top of the generated swift file (for
   storyboards) using the `import` flag. Multiple imports can be added by repeating
   this flag.  
-  [David Jennes](https://github.com/djbe), [#175](https://github.com/AliSoftware/SwiftGen/pull/175) 
+  [David Jennes](https://github.com/djbe), [#175](https://github.com/AliSoftware/SwiftGen/pull/175)
 * Remove the `key` param label from the `tr` function for Localized String in the swift3 template [AndrewSB](https://github.com/AndrewSB), [#190](https://github.com/AliSoftware/SwiftGen/pull/190)
+* The `swiftgen images` command now uses the `actool` utility to parse asset catalogs,
+  ensuring that the parser correctly handles namespaced folders.
```

Comment:
I thought I copied my previous changelog, seems I didn't, or Xcode is messing with me.